regcore_pgsql/views.py

```diff
@@ -20,6 +22,11 @@ def matching_sections(search_args):
     if search_args.regulation:
         sections_query = sections_query.filter(
             documentindex__doc_root=search_args.regulation)
+
+    notice = Notice.objects.filter(document_number=OuterRef('version'))
```

Comment:
This needs to be ordered with `-effective_on`

Reply:
make this into it's own function